it/controllers/employment/EmploymentDetailsAndBenefitsControllerISpec.scala

```diff
@@ -25,7 +25,7 @@ import utils.{IntegrationTest, ViewHelpers}
 
 
 class EmploymentDetailsAndBenefitsControllerISpec extends IntegrationTest with ViewHelpers {
-
+val taxYearEOY=taxYear -1
```

Comment:
Theres a couple of other places where we use "taxYear - 1" within this file (lines 52 and 62), might be worth replacing them with your new val